snippets/logging-loki-vs-lokistack-snip.adoc

```diff
@@ -0,0 +1,9 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+In logging subsystem documentation, LokiStack refers to the logging subsystem supported combination of Loki, and web proxy with OpenShift Container Platform authentication integration. LokiStack’s proxy uses OpenShift Container Platform authentication to enforce multi-tenancy. Loki refers to the log store as either the individual component or an external store.
```

Comment:
Nitpick: If Loki and LokiStack are terms from the logs, shouldn't they be formatted as `LokiStack` and `Loki` instead?

Reply:
Loki and LokiStack are names of some objects, but are also concepts, so in this context, no.

Reply:
got it!